### DISCUSSION TITLE
Après le vétérinaire, l'infirmier royal

### DISCUSSION THREAD

**Discussion by Gabrielre14** [Crownicles/Crownicles#1850] [Idées acceptées / déjà implémentées]
Je viens d'avoir une nouvelle idée pour la magasin des missions, avec l'augmentation de l'importance des combats. 
Que pensez-vous, après le vétérinaire de la cour, d'un "infirmier royal", dont on peut acheter une consultation afin de régénérer nos points de combats (⚡) pour 10 gemmes ? 
A cela pourrait s'ajouter un petit gain de vie, voire un soin d'altération d'état (de type étourdi, gelé, ivre, malade, blessé ou K.O.) pour un prix plus élevé (en attendant une dizaine ou vingtaine de minutes le temps de cette consultation). A voir pour l'équilibrage...

Je mets mes pensées à l'écrit ici afin d'éviter de les oublier ^^

**Comment by BastLast** [comment]
techniquement d'autres moyens ont été ajoutés pour regen ses points de combat je classe en déjà implémenté
